### DISCUSSION TITLE
is the glibc vram crash fixed?

### DISCUSSION THREAD

**Discussion by Amgsdfyhadfgybi** [vinegarhq/sober#139] [Q&A]
i know discussions is literally dead and im not sure why they dont have a question option in the issues tab, but im starting to think that the crash that sometimes happens when youre playing games that demand a lot of your vram is fixed, is it really though?

**Comment by Vlad55432** [comment]
nope, still noticing these crashes. Strange thing, that it's only happens on certain games and dev don't know what to do
https://discord.com/channels/1069506340973707304/1099853933712773193/1276860617390690306

**Reply by 41chemist** [reply]
When using this link, nothing in discord is loaded. 
![Screenshot from 2024-08-28 15-57-52](https://github.com/user-attachments/assets/a58664a9-8179-4f27-8171-fb6182ab27ec)

**Comment by adverbialsatz** [comment]
Fixed now
